ext/storage_sources/s3_store/s3_log_system.cpp

```diff
@@ -0,0 +1,71 @@
+#include <aws/core/Aws.h>
+#include <aws/core/utils/logging/AWSLogging.h>
+#include <aws/core/utils/logging/LogSystemInterface.h>
+
+#include "s3_log_system.h"
+
+#include <cstdarg>
+
+S3LogSystem::S3LogSystem(WT_EXTENSION_API *wtApi, uint32_t wtVerbosityLevel)
+{
+    // Mapping the desired WiredTiger extension verbosity level to a rough equivalent AWS SDK
+    // verbosity level.
+    static const std::map<int32_t, Aws::Utils::Logging::LogLevel> verbosityMapping = {
+      {-3, Aws::Utils::Logging::LogLevel::Error}, {-2, Aws::Utils::Logging::LogLevel::Warn},
+      {-1, Aws::Utils::Logging::LogLevel::Info}, {0, Aws::Utils::Logging::LogLevel::Info},
+      {1, Aws::Utils::Logging::LogLevel::Debug}};
+    if (verbosityMapping.find(wtVerbosityLevel) != verbosityMapping.end()) {
```

Comment:
We currently do not allow for changing the verbosity level because we are getting the verbosity level from the config that gets set at the beginning.